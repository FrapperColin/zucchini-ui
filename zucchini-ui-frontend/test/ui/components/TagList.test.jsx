import React from "react";
import renderer from "react-test-renderer";
import { BrowserRouter, Route } from "react-router-dom";

import TagList from "../../../src/ui/components/TagList";

describe("TagList", () => {
  it("should contain labels with tags", () => {
    const tags = ["A", "B", "C"];
    const testRunId = "sampleId";

    const component = renderer.create(
      <BrowserRouter>
        <TagList testRunId={testRunId} tags={tags} />
      </BrowserRouter>
    );

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
