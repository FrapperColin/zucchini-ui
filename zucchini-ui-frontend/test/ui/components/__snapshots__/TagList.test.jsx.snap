// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TagList should contain labels with tags 1`] = `
<span>
  <a
    className="label label-info"
    href="/test-runs/sampleId/tag-details"
    onClick={[Function]}
  >
    @
    A
  </a>
   
  <a
    className="label label-info"
    href="/test-runs/sampleId/tag-details"
    onClick={[Function]}
  >
    @
    B
  </a>
   
  <a
    className="label label-info"
    href="/test-runs/sampleId/tag-details"
    onClick={[Function]}
  >
    @
    C
  </a>
  
</span>
`;
